two_factor/plugins/phonenumber/models.py

```diff
@@ -8,11 +8,13 @@
 from django_otp.util import hex_validator, random_hex
 from phonenumber_field.modelfields import PhoneNumberField
 
-from two_factor.gateways import make_call, send_sms
+from two_factor.gateways import make_call, send_sms, send_whatsapp
 
+WHATSAPP = 'wa'
```

Comment:
Not sure what this is for, it's only used in a few places. Also why are we using `"wa"` rather than `"whatsapp"`?

Reply:
explicit is better. agreed.